### ISSUE TITLE
Unexpected addresses using Range search + error with byte search

### ISSUE THREAD

**Issue by testitm** [Qwokka/Cetus#46]
Hey!
Sorry to bother you! I found your project very interesting and spent a lot of time debugging an online game for fun.

But I found some problems with Cetus.
1) Range search.  
You can see on the screenshot, that address in search results is not bounded by Range values. 
<img width="500" alt="Screenshot 2022-09-06 at 22 06 17" src="https://user-images.githubusercontent.com/87523283/188719969-2fd05428-5cc9-4e11-9b98-92279d729840.png">

2) Byte search
Also, when I tried to find some bytes, I got an error
```
Uncaught TypeError: Cannot read properties of undefined (reading 'count')
    at cetus.js:821:47
(anonymous) @ cetus.js:821
```
<img width="500" alt="Screenshot 2022-09-06 at 22 07 04" src="https://user-images.githubusercontent.com/87523283/188720764-d05af13c-2977-483e-9760-426eca3530d2.png">

<img width="500" alt="Screenshot 2022-09-06 at 22 09 47" src="https://user-images.githubusercontent.com/87523283/188720343-31cbb55f-4428-4d19-9f35-56fd6c07edaa.png">
<img width="500" alt="Screenshot 2022-09-06 at 22 08 51" src="https://user-images.githubusercontent.com/87523283/188720335-97284174-1eea-4ace-be54-c446fc034224.png">

Maybe I didn't get something, but these two bugs make it hard to edit the certain addresses that I found in the Memory Viewer :)

**Comment by testitm** [comment]
Oh, the Range problem depends on the value type. So Range works as desired when I chose i8 instead of i32.

**Comment by Qwokka** [comment]
Hey @testitm 

Thanks for catching this! I've confirmed this and I see why it's going on. Unfortunately I don't have time to fix it this evening. I'm hoping to have it fixed tomorrow.

Jack

**Comment by testitm** [comment]
No problem, thank you anyway :)
Great tool, I will try to fix some UI problems in future, I hope :)

**Comment by Qwokka** [comment]
I think this is now fixed with fab16cb9908f10dc8de1a42a20a780b9d52b7b73

Please let me know if you have more issues. Thanks again for pointing this out!
Jack

**Comment by testitm** [comment]
Hey!
Found it broken https://github.com/Qwokka/Cetus/commit/fab16cb9908f10dc8de1a42a20a780b9d52b7b73#r84304655

**Comment by Qwokka** [comment]
Whoops, this should be fixed with de68f098b4f441575606a9582e00b687edab6e29
